File: viajerosrepoangular-master/viajerosrepoangular-master/viajeros/src/app/models/Payments/PreferenceTripDto.ts
export class PreferenceTripDto {
    idviaje: number;
    monto: number;
    title: string;
    description: string;
    idpasajero: number;
  
    constructor(idviaje: number, monto: number, title: string, description: string, idpasajero: number) {
      this.idviaje = idviaje;
      this.monto = monto;
      this.title = title;
      this.description = description;
      this.idpasajero = idpasajero;
    }
  }
  

File: viajerosrepojava-master/viajerosrepojava-master/viajeros/viajeros/src/main/java/ps/com/viajeros/dtos/payments/PreferenceTripDto.java
package ps.com.viajeros.dtos.payments;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.math.BigDecimal;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class PreferenceTripDto {
    private Long idviaje;
    private BigDecimal monto;
    private String title;
    private String description;
    private Long idpasajero;
}
